test/integration/admin/api_docs/account_api_docs_controller_test.rb

```diff
@@ -33,6 +33,25 @@ class ProviderLoggedInTest < Admin::ApiDocs::AccountApiDocsControllerTest
       assert_same_elements expected_api_docs_data, actual_api_docs_data
     end
 
+    test 'index has the API column' do
+      service_2 = FactoryGirl.create(:simple_service, account: provider)
+      FactoryGirl.create(:api_docs_service, service: service, account: provider)
+      FactoryGirl.create(:api_docs_service, service: service_2, account: provider)
+      get admin_api_docs_services_path
+
+      assert_xpath "//*[@id='content']/table/thead/th[4]", 'API' # Name of the column
```

Comment:
Well I think all those xpath would be more obvious as a cucumber

Reply:
![image](https://user-images.githubusercontent.com/11318903/46944424-4e1a5d80-d072-11e8-9a44-4f36713d66cf.png)